.github/workflows/dvc-diff.yml

```diff
@@ -0,0 +1,53 @@
+# Checks for image diffs in a Pull Request and adds a GitHub comment showing the diff
+name: DVC image diff
+
+on:
+  pull_request:
+    paths:
+      - 'pygmt/tests/baseline/*.png.dvc'
```

Comment:
Is there a way to get this workflow to run only on commits where `*.png.dvc` files have changed? The dvc-diff workflow still appears to run on commits where no *.png.dvc files have changed, e.g. at e14674a786eedf82a6fe6a0d9eb7cb69f9f26d0f.